File: architecture/OC/Demo/Demo/ViewController.m
//
//  ViewController.m
//  Demo
//
//  Created by CQCA202121101_2 on 2025/12/5.
//

#import "ViewController.h"
#import "BViewController.h"

@interface ViewController ()
@property (nonatomic ,strong) UIViewController *vc;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
//    self.vc = [BViewController new];
//    [self.navigationController pushViewController:self.vc  animated:YES];
    // Do any additional setup after loading the view.
}


- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    [super touchesBegan:touches withEvent:event];
    [self.navigationController pushViewController:[BViewController new]  animated:YES];

    
}

@end

File: architecture/OC/Demo02/Demo02/ViewController.swift
//
//  ViewController.swift
//  Demo02
//
//  Created by CQCA202121101_2 on 2025/12/5.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        super.touchesBegan(touches, with: event)
        self.navigationController?.pushViewController(TestVC(), animated: true)
    }

}

